pkg/services/alerting/notifier.go

```diff
@@ -34,14 +34,15 @@ type notificationService struct {
 	renderService rendering.Service
 }
 
-func (n *notificationService) SendIfNeeded(context *EvalContext) error {
+func (n *notificationService) SendIfNeeded(context *EvalContext) {
 	notifierStates, err := n.getNeededNotifiers(context.Rule.OrgID, context.Rule.Notifications, context)
 	if err != nil {
-		return err
+		n.log.Error("Failed to get alert notifiers", "error", err)
```

Comment:
Good that you added error logging here since there's currently no handling of the error returned :+1: